k8sClient.py

```diff
@@ -37,19 +43,97 @@ def get_nodes_with_label(self, label_selector: str) -> list[str]:
             for e in self._client.list_node(label_selector=label_selector).items
         ]
 
+    @staticmethod
+    def _oc_get_cmd(cmd: str | Iterable[str]) -> list[str]:
```

Comment:
These names are kind of confusing since "get" is a special keyword for k8s.

Reply:
renamed.